src/components/Dashboard/FilingHistoryList/filings/AlterationFiling.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <FilingTemplate class="alteration-filing" :filing="filing" :index="index">
+    <template #subtitle>
+      <FiledAndPendingPaid
+        v-if="isFutureEffectivePending"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
+
+      <FutureEffectivePaid
+        v-else-if="isFutureEffective"
+        class="item-header-subtitle"
+        :filing="filing"
+        :index="index"
+      />
+    </template>
```

Comment:
Here we check 2 conditionals for the "subtitle" slot. If both are false the the fallback (common/generic) slot content is used.